python/rmm/_lib/memory_resource.pyx

```diff
@@ -57,8 +67,10 @@ cdef extern from "rmm/mr/device/managed_memory_resource.hpp" \
 cdef extern from "rmm/mr/device/cuda_async_memory_resource.hpp" \
         namespace "rmm::mr" nogil:
     cdef cppclass cuda_async_memory_resource(device_memory_resource):
-        cuda_async_memory_resource(optional[size_t] initial_pool_size,
-                                   optional[size_t] release_threshold) except +
+        cuda_async_memory_resource(
+            optional[size_t] initial_pool_size,
+            optional[size_t] release_threshold,
+            optional[cudaMemAllocationHandleType] export_handle_type) except +
```

Comment:
Problem: this third parameter type only exists if compiled with CUDA 11.2 or greater. How do I handle this?

Reply:
> Problem: this third parameter only exists if compiled with CUDA 11.2 or greater. How do I handle this?

could the third argument be added here? 

https://github.com/rapidsai/rmm/pull/1030/files#diff-27f94e026227c9d329cb90cbcfae81c25c829d1c49d84c7491ea71f4ac2041c5R114.